test/sagemaker_tests/huggingface/inference/integration/sagemaker/test_hub_model.py

```diff
@@ -29,32 +26,32 @@
 @pytest.mark.model("tiny-distilbert")
 @pytest.mark.processor("cpu")
 @pytest.mark.cpu_test
-def test_sm_trained_model_cpu(sagemaker_session, framework_version, ecr_image, instance_type):
+def test_hub_model_cpu(sagemaker_session, framework_version, ecr_image, instance_type, region):
     instance_type = instance_type or "ml.m5.xlarge"
     try:
         _test_hub_model(sagemaker_session, framework_version, ecr_image, instance_type, model_dir)
-    except UnexpectedStatusException as e:
-        dump_logs_from_cloudwatch(e)
+    except Exception as e:
+        dump_logs_from_cloudwatch(e, region)
         raise
 
 
 @pytest.mark.model("tiny-distilbert")
 @pytest.mark.processor("gpu")
 @pytest.mark.gpu_test
-def test_sm_trained_model_gpu(sagemaker_session, framework_version, ecr_image, instance_type):
+def test_hub_model_gpu(sagemaker_session, framework_version, ecr_image, instance_type, region):
     instance_type = instance_type or "ml.p2.xlarge"
     try:
         _test_hub_model(sagemaker_session, framework_version, ecr_image, instance_type, model_dir)
-    except UnexpectedStatusException as e:
-        dump_logs_from_cloudwatch(e)
+    except Exception as e:
+        dump_logs_from_cloudwatch(e, region)
         raise
 
 
 def _test_hub_model(sagemaker_session, framework_version, ecr_image, instance_type, model_dir, accelerator_type=None):
     endpoint_name = sagemaker.utils.unique_name_from_base("sagemaker-huggingface-serving-hub-model")
 
     env = {
-        "HF_MODEL_ID": "sshleifer/tiny-distilbert-base-uncased-finetuned-sst-2-english",
+        "HF_MODEL_ID": "philschmid/tiny-distilbert-classification",
```

Comment:
Is there a way we could remove GitHub ID from this and use something more generic?

Reply:
What do you mean with Github ID and generic? It is a repository on hf.co/models https://huggingface.co/philschmid/tiny-distilbert-classification, which is maintained by me and which should be tested in the test